include/universal/utility/generateClosurePlots.hpp

```diff
@@ -0,0 +1,450 @@
+#ifndef GENERATECLOSUREPLOTS_H
```

Comment:
replace the compilation guards with a simple
```
#pragma 
```

it is a more modern way to accomplish single inclusion.